### ISSUE TITLE
the issue of attention networks

### ISSUE THREAD

**Issue by 18370029656** [HighwayWu/ImageInpainting#11]
Hello, I would like to ask about the issue of attention networks. In the attention.py, what are the generated and known values of the input network in line 20? Are the image features of unknown and known regions?
![Screenshot 2023-04-17 160502](https://user-images.githubusercontent.com/108872700/232423650-ceb586dc-6012-4af2-b2c1-d42771fd348c.png)

**Comment by HighwayWu** [comment]
Yes. The "generated" means the inpainting regions, where the features are generated by network. The "known" means the existing regions, like background.

**Comment by 18370029656** [comment]
Hello, since 'generate' refers to the inpainting regions, why do lines 27 and 31 still need to determine flag==1 and flag==0. May I ask what the purpose of these two lines is??
![image](https://user-images.githubusercontent.com/108872700/233589262-cefa9433-cf7c-49e4-857d-c10d52837e0c.png)

**Comment by Runing-Lab** [comment]
Hello, have you solved it? I also have an error here when I batchsize=2
